config/initializers/raven.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'raven'
+
+if %w[production].include?(Rails.env) && ENV['SENTRY_DSN'].present?
+  Raven.configure do |config|
+    config.dsn = ENV['SENTRY_DSN']
+    config.environments = %w[production]
+    config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+  end
+end
```

Comment:
Our setup means all our server instances are production environments, so...
```suggestion
if Rails.env.eql?('production') && ENV['SENTRY_DSN'].present?
  Raven.configure do |config|
    config.dsn = ENV['SENTRY_DSN']
    config.environments = Rails.env
    config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
  end
end
```

Reply:
👍